collateral/deployment/k8s-without-helm-reference.md

```diff
@@ -0,0 +1,53 @@
+# Deploying Sourcegraph to Kubernetes (k8s) without Helm
+
+## Introduction
+This material is intended to be used as a handout to customers working with the Customer Engineering team. The materials are meant to serve as an overview and resource to aid in deep diving into Sourcegraph's documentation and common deployment tasks.
```

Comment:
```suggestion
This material is intended to be used as a handout to customers working with a Sourcegraph Customer Engineer to deploy their own instance. The materials are meant to serve as an overview and resource to aid in deep diving into Sourcegraph's documentation and common deployment tasks.
```

Clarifying here to add SG explicitly + confirm the task at hand, since we do lots of stuff